.travis.yml

```diff
@@ -61,6 +61,11 @@ matrix:
     - SQLITE_DATABASE_URL=/tmp/test.db
     script:
     - (cd diesel_cli && travis-cargo test -- --no-default-features --features "sqlite-bundled")
+  - rust: 1.24.1
+    env:
+    - ENSURE_SUPPORTED_RUST_VERSION=1.24.1
```

Comment:
Does this do anything ?

Reply:
Puts a nice label in the matrix so it's clear what this does

Reply:
Oh :+1: